modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -329,15 +329,18 @@
 	ckeywhitelist = list ("especiallystrange")
 
 /obj/item/storage/box/large/custom_kit/bibledog/PopulateContents()
-	new /obj/item/gun/ballistic/automatic/pistol/m1911/bibledog(src)
+	new /obj/item/gun/ballistic/automatic/pistol/m1911/bibledog(src) //We can't expect god do all the works
 	new /obj/item/ammo_box/magazine/m45(src)
 	new /obj/item/ammo_box/magazine/m45(src)
 	new /obj/item/clothing/under/f13/bibledog(src)
-	new /obj/item/ammo_box/magazine/m45(src)
-	new /obj/item/clothing/suit/armor/f13/combat/swat(src)
+	new /obj/item/clothing/mask/mummy/bibledog(src)
+	new /obj/item/clothing/suit/armor/f13/combat/swat/bibledog(src)
 	new /obj/item/storage/belt/holster/legholster(src)
+	new /obj/item/melee/classic_baton/police(src)
 	new /obj/item/book/granter/trait/bibledog(src) //being able to use the bible
```

Comment:
This does nothing to let you use the bible and instead gives you anti-magic immunity. That's, uh. No.
```suggestion
```